packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js

```diff
@@ -22,6 +22,7 @@ const GitHubApi = require( '@octokit/rest' );
 const PACKAGE_JSON_TEMPLATE_PATH = require.resolve( '../templates/release-package.json' );
 const BREAK_RELEASE_MESSAGE = 'You aborted publishing the release. Why? Oh why?!';
 const NO_RELEASE_MESSAGE = 'No changes for publishing. Why? Oh why?!';
+const AUTH_REQUIRED = 'This command requires you to be logged in.';
```

Comment:
Come on, we're humans! :D 

```suggestion
const AUTH_REQUIRED = 'You must be logged to execute this command.';
```